README.md

```diff
@@ -47,6 +47,10 @@ For example, if you place a `metadata.yaml` file within your app repository, the
 my-big-map-app:
   metadata:
     $ref: https://github.com/my-org/my-big-map-app/raw/main/metadata.yaml
+  categories:
+    - technology-aiida
+    - technology-ase
+    - quantum
```

Comment:
Please add one wp category to the examples so that is clear that those should be applied.